r/R/dplyr-funcs-math.R

```diff
@@ -80,4 +80,21 @@ register_bindings_math <- function() {
       options = list(ndigits = digits, round_mode = RoundMode$HALF_TO_EVEN)
     )
   })
+  
+  register_binding("sqrt", function(x, ...) {
+    # accepts and ignores ... for consistency with base::trunc()
```

Comment:
I think you can drop `...` from the signature and these comments because the `base::` versions of these particular functions don't take dots:

```
> sqrt
function (x)  .Primitive("sqrt")
> exp
function (x)  .Primitive("exp")
```

Reply:
Thanks Neal - fixed!